translations/ES_eu (international Spanish really)

```diff
@@ -0,0 +1,128 @@
+            - |
+              Las comisiones bajas y las confirmaciones rápidas han vuelto con Bitcoin Cash. La red está creciendo de nuevo. Usuarios, comerciantes, empresas e inversores están creando el futuro con dinero de usuario a usuario de verdad.
+        - question: "¿Por qué Bitcoin no aumentó el tamaño del bloque si era tan sencillo?"
+          answer:
+            - |
+              Algunos de los desarrolladores no entendían ni estaban de acuerdo con la visión inicial de dinero electrónico de usuario a usuario que creó Satoshi Nakamoto. <a href="https://medium.com/@jgarzik/bitcoin-is-being-hot-wired-for-settlement-a5beb1df223a" target="_blank">En cambio, optaron por que Bitcoin se convirtiera en una plataforma de liquidación</a>.
+            - |
+              Muchos mineros y usuarios confiaron en estos desarrolladores, mientras que otros se dieron cuenta de que estaban llevando a la comunidad por un camino diferente del esperado.
+            - |
+              Estas dos visiones tan diferentes de Bitcoin son fundamentalmente incompatibles y llevaron a la división de la comunidad.
+        - question: "¿Acaso no aumentó SegWit la capacidad del bloque? ¿Tendrá Bitcoin Cash SegWit en algún momento?"
+          answer:
+            - |
+              Segregated Witness o "SegWit" es un sustituto inaceptable al aumento del tamaño del bloque por varias razones.
+            - |
+              En primer lugar, incluso si se utilizara en el 100% de las transacciones, el incremento solo llevaría a bloques de 1,7MB. Por lo tanto, en el mejor de los casos esto sólo constituye un pequeño aumento y no permite crecimiento exponencial o uso global. En segundo lugar, esta implementación mediante <i>soft fork</i> da lugar a firmas descartables, las cuales <a href="https://bitcrust.org/blog-incentive-shift-segwit" target="_blank">debilitan el modelo de seguridad de Bitcoin.</a> En tercer lugar, hace que los aumentos de capacidad futuros sean más difíciles de implementar debido a ineficiencias en el uso del ancho de banda y a ataques de hashing cuadrático. SegWit no resuelve este asunto ya que un atacante no se ve obligado a utilizarlo.
+            - |
+              Estas y otras razones fueron la causa de que fuese necesario implementar Bitcoin Cash como bifurcación previa a la implementación de SegWit. Bitcoin Cash no implementará SegWit.
+        - question: "¿Cuál es el equipo a cargo del desarrollo de Bitcoin Cash?"
+          answer:
+            - |
+              A diferencia de la situación de Bitcoin descrita arriba, en Bitcoin Cash no existe un único equipo de desarrollo. Existen múltiples equipos independientes.
+            - |
+              Esta descentralización del desarrollo (y de las implementaciones de software) constituye un paso adelante muy importante y necesario.
+        - question: "¿Cuál es el símbolo o etiqueta de Bitcoin Cash?"
```

Comment:
I think "símbolo bursátil" is in order here, since it's more precise and the direct translation for "ticker".

Reply:
I'm really not happy with either of those two options, to be honest.
How about "la abreviatura"? 

Símbolo bursátil sounds way off to me in this context, remember this is addressed to the mainstream public; símbolo bursátil sounds very specific to stocks and trading, which have nothing to do with bitcoin (a currency, not simply an "investment"). Do I make sense?